src/bin/train.cc

```diff
@@ -456,49 +480,74 @@ int main(int argc, char * argv[]) {
         trainer->update(lr_decay);
 
         num_trained_samples += batch.source_ids[0].size();
+        num_trained_words += batch.target_ids.size() * batch.target_ids[0].size();
         if (!train_sampler.hasSamples()) {
           train_sampler.rewind();
         }
 
-        const string fmt_str = "Trained: batch=%d samples=%d lr=%.6e";
+        const string fmt_str = "Trained: batch=%d samples=%d current-proc-words=%d lr=%.6e";
```

Comment:
current-proc-words -> words (maybe short name is better because of reducing log file size)